packages/esm-patient-flags-app/src/flags/flags-risk-count-extension/extension-config-schema.ts

```diff
@@ -0,0 +1,15 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const riskCountExtensionConfigSchema = {
+  hideOnPages: {
+    _type: Type.Array,
+    _default: ['Patient Summary'],
```

Comment:
```suggestion
    _default: ['Patient Summary'],
    _description:
      'List of page names where the risk count badge should not be displayed. Matches against the last segment of the URL path (e.g., "Patient Summary", "Vitals", "Medications").',
```

Or something to that effect.